tests/test_SklearnGLMClassifierConverter.py

```diff
@@ -156,9 +162,13 @@ def test_model_logistic_regression_multi_class_liblinear_l1(self):
     @unittest.skipIf(not onnx_built_with_ml(),
                      reason="Requires ONNX-ML extension.")
     def test_model_logistic_regression_multi_class_saga_elasticnet(self):
-        model, X = self._fit_model_multiclass_classification(
-            linear_model.LogisticRegression(
-                solver='saga', penalty='elasticnet', l1_ratio=0.1))
+        if StrictVersion(sklearn.__version__) < StrictVersion('0.21.0'):
```

Comment:
The previous code only works for sklearn >= 0.21 ?

Reply:
l1_ratio and elasticnet penalty for LogiticRegression got added in 0.21.0, hence the check.